core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -66,6 +66,7 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-mapperXmx", conf.mapperXmx,
       "-output", conf.HDFSOutputDir.get,
       "-J", "-log_level", "-J", conf.h2oNodeLogLevel,
+      "-J", "-disable_web",
```

Comment:
could we have an option to handle that? Or in fact a way how to pass options to external cluster launcher. We can always stick with `-Dai.h2o.disable_web` syntax...

Reply:
We can have option for sure 👍 I wanted to mimic the internal cluster behaviour, where we simply disable it without no options for security reasons